### ISSUE TITLE
Pagination for /updates is upset by joins

### ISSUE THREAD

**Issue by owtaylor** [fedora-infra/bodhi#4130]
If you specify filter parameters that cause joins, then the pagination in for /updates works weirdly - e.g., right now

  https://bodhi.fedoraproject.org/updates/?content_type=flatpak&page=1&rows_per_page=5

has 4 items. And if you try to iterate through all the items by using page 1....(total/rows_per_page), you don't get them everything, and you can get repeated items. (A workaround to get everything would be to increase the page until you get an empty result, and handle duplicates.)

Basically the pagination is happening is happening on the raw query, then deduplication on primary key ID happens. (https://docs.sqlalchemy.org/en/13/faq/sessions.html#faq-query-deduplicating)

I don't have enough sqlalchemy knowledge to guess at a good solution - with raw SQL using subqueries instead of joins would be possible.

**Comment by mattiaverga** [comment]
Fixed with #5753
